README.md

```diff
@@ -1 +1,3 @@
-# review-p-trial
+# Cristian - Teddy Project
+
+## This is the Collaboration Trial Project between Cristian Camilo and Teddy-Livingstone. The project is a Tribute Page of Norman Borlaug
```

Comment:
- [ ] Please do make sure to add a more descriptive README file. You can use the Microverse README template. It can be found [here](https://github.com/microverseinc/readme-template)